app/models.py

```diff
@@ -57,14 +56,6 @@ def load_user(id):
     return Users.query.get(int(id))
 
 
-# Sizes enum for companies.company_size
-class Sizes(enum.Enum):
```

Comment:
😢

Reply:
So annoying we can't migrate this .. 

First time I discover something about Postgres that is not cool haha